### DISCUSSION TITLE
Label search doesn't work with emojis

### DISCUSSION THREAD

**Discussion by benjaminleonard** [community/community#5146] [Projects and Issues] [answered]
If a label has an emoji in the name, it seems like it doesn't come up in the view filter/search.

<img width="482" alt="image" src="https://user-images.githubusercontent.com/4020798/129712770-f2c08b70-0ff1-4aab-b11a-f563aee1d757.png">

<img width="223" alt="image" src="https://user-images.githubusercontent.com/4020798/129712794-486220d9-62f6-4585-b733-dafa33a4ca52.png">

If I remove the emoji, and search for the label it appears fine.

**Comment by Simran-B** [comment, marked as answer]
Looks like you are referring to the issues beta? It seems to work fine in the regular issue search. Does your search string contain a space between emoji but the label does not have one?

**Reply by benjaminleonard** [reply]
Thanks for the advice.

Yes, it is the issues beta. It does work for me in the regular issue search – though I think due to the rendering of the emojis you can't copy the tag, it has to use the emoji name. So I got a match with `label:":black_square_button: cpu"`.

But unfortunately that doesn't work in the beta either.

**Reply by Simran-B** [reply]
https://github.blog/changelog/2021-08-17-the-new-github-issues-08-17-update/ mentions this:

> Fixed an issue with filtering against fields which contained emojis.

Filtering is different to searching, though, right?

**Reply by benjaminleonard** [reply]
Oh that sounds like it should fix it, but unfortunately not. It's strange because the label appears in the autocomplete, but as soon as it's selected it doesn't return the corresponding issue.

**Reply by Simran-B** [reply]
Maybe it's fixed now?

> Bug fix to resolve issues when filtering over labels which contained `:`.

https://github.blog/changelog/2021-09-03-the-new-github-issues-09-03-update/

**Reply by benjaminleonard** [reply]
Yes! Fixed

Thanks for your help 😄
